building-srpm-in-copr/README.md

```diff
@@ -90,3 +85,27 @@ copr_client.build_proxy.create_from_custom(ownername="lbarczio", projectname="og
                            script_builddeps=["git", "packit", "python3-wheel", "python3-pip", "python3-setuptools",
                             "python3-setuptools_scm", "python3-setuptools_scm_git_archive"])
 ```
+
+### Plan how to build SRPMs in Copr in Packit Service
+
+- create CLI command or only Packit API method for preparing sources (would mostly use existing functionality, described above)
+- implement creation of the script (Python/bash - Python would be easier for handling with configs)
+  - script will be created dynamically each time
+  - pass the needed arguments and use the method/command
+- implement using the script for creating SRPMS for Copr builds
```

Comment:
would it make sense adding it inside packit?

Reply:
not sure about this, I think it makes more sense to have it only in p-s (I am thinking whether there could be any usage only in Packit), but will look into the code and rethink it

Reply:
Packit tool will be installed anyways, or?  From my point of view, after debugging similar stuff countless times, having it in some tool is always better...   there's nothing worse than logic hard-wired to a service without an easy way to reproduce the build process locally (long live MBS).